app/assets/javascripts/discourse/app/lib/notification-items/liked.js

```diff
@@ -30,9 +22,17 @@ export default class UserMenuLikedNotificationItem extends UserMenuNotificationI
 
   get labelWrapperClasses() {
     if (this.count === 2) {
-      return "double-user";
+      return ["double-user"];
     } else if (this.count > 2) {
-      return "multi-user";
+      return ["multi-user"];
     }
   }
+
+  get count() {
+    return this.notification.data.count;
+  }
+
+  get username2() {
```

Comment:
```suggestion
  get #username2() {
```

Just wondering if it needs to be public.